modules/core/src/main/java/org/apache/ignite/internal/commandline/Command.java

```diff
@@ -51,6 +53,29 @@ public static GridClient startClient(GridClientConfiguration clientCfg) throws E
         return client;
     }
 
+    /**
+     * @return Cluster information to show user for.
+     */
+    default String getFullClusterInfo(GridClientClusterState clientCfg){
+        String clusterName = getClusterInfo(clientCfg);
+        String clusterInfo = "Cluster";
+        if (clusterName != null)
+            clusterInfo = "Cluster \"" + clusterName + "\"";
+        return clusterInfo;
+    }
+
+    /**
+     * @return Cluster information to show user for.
+     */
+    default String getClusterInfo(GridClientClusterState clientCfg){
+        String clusterName = null;
+        try{
+            clusterName = clientCfg.clusterName();
+        } catch (GridClientException ignored){
```

Comment:
Try-catch block doesn't comply with our code style. Also there should be some more blank lines. Check out this link: https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines